Dockerfile.cpu

```diff
@@ -22,14 +22,12 @@ ENV LD_PRELOAD="/usr/lib/x86_64-linux-gnu/libtcmalloc_minimal.so.4:/usr/local/li
 
 RUN echo 'ulimit -c 0' >> ~/.bashrc
 
-RUN pip install intel_extension_for_pytorch==2.5.0
+RUN pip install intel_extension_for_pytorch==2.5.0 # FIXME: should this be a dependency in requirements-cpu.txt?
 
 WORKDIR /workspace
 
-ARG PIP_EXTRA_INDEX_URL="https://download.pytorch.org/whl/cpu"
-ENV PIP_EXTRA_INDEX_URL=${PIP_EXTRA_INDEX_URL}
 RUN --mount=type=cache,target=/root/.cache/pip \
-    --mount=type=bind,src=requirements-build.txt,target=requirements-build.txt \
+    --mount=type=bind,src=requirements-build.txt,target=requirements-build.txt \{cpu,cuda,hpu,neuron,openvino,tpu,xpuo
```

Comment:
Is this a typo?

Reply:
Yes, botched merge conflict again, fixed.